Cargo.toml

```diff
@@ -36,6 +36,7 @@ signatory-dalek = "0.10"
 signatory-yubihsm = { version = "0.10", optional = true }
 subtle-encoding = "0.2"
 x25519-dalek = { version = "0.3", default-features = false, features = ["std", "u64_backend"] }
+iq-bech32 = "0.1"
```

Comment:
FYI, I'm merging this crate into subtle-encoding (it has the same API).

This is fine for now though, and I'll bump it when I cut a subtle-encoding v0.3.0 release.